### PR TITLE
Include a clientid in mqtt commands

### LINKED ISSUE #42: MQTT spec does not allow slashes and underscores
Hi, after the latest update of Mosquitto broker (MQTT) in Home Assistant I am not receiving any data. It seem that MQTT no longer allows slashes and underscores in client identifier. Please help.....

### PR DESCRIPTION
**Pull request by RSd108**
A configurable clientid is added to mqtt config

This fixes  #42  and #43

### CODE REVIEW

**Review by TaariqSurtee — APPROVED** [review]

**Comment by TaariqSurtee** [comment]
It looks correct. I will have to test it.

File: sources/inverter-mqtt/mqtt-push.sh [hunk 1]
```diff
@@ -8,12 +8,14 @@ pushMQTTData () {
     MQTT_DEVICENAME=`cat /etc/inverter/mqtt.json | jq '.devicename' -r`
     MQTT_USERNAME=`cat /etc/inverter/mqtt.json | jq '.username' -r`
     MQTT_PASSWORD=`cat /etc/inverter/mqtt.json | jq '.password' -r`
+	MQTT_CLIENTID=`cat /etc/inverter/mqtt.json | jq '.clientid' -r`
```
Comment (RHeynsZa) [inline sources/inverter-mqtt/mqtt-push.sh:11]: This is a tab, but the file uses spaces

File: config/mqtt.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
     "devicename": "voltronic",
     "username": "",
     "password": "",
+	"clientid": "voltronic_bd8041d0cdf131a6ba4e5b3360b8bc5a",
```
Comment (RHeynsZa) [inline config/mqtt.json:8]: tab vs spaces

**Review by RHeynsZa — CHANGES_REQUESTED** [review]

**Comment by RHeynsZa** [comment]
Tested this, still not sending mqtt messages

**Comment by RHeynsZa** [comment]
> It looks correct. I will have to test it.

I tested on my side, this works, but seems like the underscore had some issues for my broker, so i just set the client to `inverter`

**Pull request merged**